chains/ethereum/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web3modal/ethereum",
-  "version": "2.0.0-beta.7",
+  "version": "2.0.0-b648e83",
```

Comment:
Meant to be 0.8 ?

Reply:
Will tag before merging 👍 this is for canary